dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1230,23 +1230,19 @@ public Workflow manualRegister(@ApiParam(hidden = true) @Auth User user,
         @ApiParam(value = "Descriptor type", required = true) @QueryParam("descriptorType") String descriptorType,
         @ApiParam(value = "Default test parameter file path") @QueryParam("defaultTestParameterFilePath") String defaultTestParameterFilePath) {
 
-        // Validate descriptor path based on type
```

Comment:
This is for line 1228, since I can't comment directly on it, and it's beyond the scope of the PR, but that one should mention nextflow, and eventually Galaxy. It's going to be an unfortunate mismatch, as the OpenAPI is generated at buildtime, the the language plugin support is at runtime.

Reply:
I agree, I haven't decided on the best way to handle these cases where a plugin at runtime approach doesn't work well. Will tackle in a follow-up.